### ISSUE TITLE
Get raw geo data [feature request]

### ISSUE THREAD

**Issue by twedl** [sharlagelfand/opendatatoronto#1]
It would be nice to be able to use boundary files straight from the API. Currently the geo data will be converted to points:

```r
library(sf)
library(geojson)
nbh_id <- "a083c865-6d60-4d1d-b6c6-b0c8a85f9c15"

res <- get_resource(nbh_id)
ggplot() + geom_sf(data = res)
# ^^ This maps points/coordinates and not polygons/boundaries.
```

This (using the internal ```get_datastore_resource``` and just not calling the `check_geometry_resource` function) can give you the polygons from the geojson:

```r
library(sf)
library(geojson)
nbh_id <- "a083c865-6d60-4d1d-b6c6-b0c8a85f9c15"

# calling internal opendatatoronto function:
res <- get_datastore_resource(nbh_id)

# convert to tibble.
res <- as_tibble(res)

# convert geometry column from geojson to SF object
sf_geo <- geojsonsf::geojson_sf(res$geometry)
res$geometry <- sf_geo$geometry

ggplot() +
  geom_sf(data = sf::st_as_sf(res, remove = FALSE, crs = 4326))
```
Depending on what else you want to do in the future, you could 

1. Add a `raw = TRUE` option to `get_resource` that doesn't process the geojson data, and add code using ```geojsonsf``` in a vignette to show how to use ```geojsonsf::geojson_sf``` to get the boundary files, or
2. Add ```geojsonsf``` (or something else) to the package itself and use it when getting a resource with format ```geojson```. [Maybe a geo person can help with a better way to convert these without adding extra dependencies.]

Good work Sharla and gl!!! 👍👍👍

**Comment by sharlagelfand** [comment]
thanks for this! i definitely intend on adding a vignette re: spatial data (as i said, not my specific strength) so it's helpful to see a use case like this.

for my own future reference, a fully functional reprex 😋 
``` r
library(opendatatoronto)
library(sf)
library(ggplot2)
library(geojsonsf)
library(geojson)

nbh_id <- "a083c865-6d60-4d1d-b6c6-b0c8a85f9c15"
res <- get_resource(nbh_id)
ggplot() + 
  geom_sf(data = res)
```

![](https://i.imgur.com/4QmwZHO.png)

``` r

res2 <- opendatatoronto:::get_datastore_resource(nbh_id)

sf_geo <- geojson_sf(res2$geometry)

res2$geometry <- sf_geo$geometry

ggplot() + 
  geom_sf(data = st_as_sf(res2, remove = FALSE, crs = 4326))
```

![](https://i.imgur.com/BY1JCxl.png)

**Comment by sharlagelfand** [comment]
This is fixed in version `0.0.0.9002` -- I didn't realize that converting the tibble to sf was replacing the existing `geometry` column, creating the points instead. 

thanks @tweed1e!

``` r
library(opendatatoronto)
library(ggplot2)

res <- get_resource("a083c865-6d60-4d1d-b6c6-b0c8a85f9c15")

ggplot() + 
  geom_sf(data = res)
```

![](https://i.imgur.com/kbZiawv.png)

<sup>Created on 2019-08-01 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>
